_posts/2017-11-23-Rust-1.22.md

```diff
@@ -0,0 +1,166 @@
+---
+layout: post
+title: "Announcing Rust 1.22"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce the latest version of Rust, 1.22.0. Rust
+is a systems programming language focused on safety, speed, and concurrency.
+
+If you have a previous version of Rust installed, getting Rust 1.22 is as easy as:
```

Comment:
Not really to do with this particular post, but I think we should change this sentence of the release announcement:

```diff
- If you have a previous version of Rust installed, getting Rust 1.X is as easy as:
+ If you have a previous version of Rust installed via `rustup`, getting Rust 1.X is as easy as:
```